src/components/intro-page/intro-page.tsx

```diff
@@ -1,27 +1,59 @@
-import React, { Fragment } from 'react'
-import { Trans, useTranslation } from 'react-i18next'
+import React, { Fragment, useEffect, useState } from 'react'
+import { Trans } from 'react-i18next'
+import { useLocation } from 'react-router'
+import { defaultFetchConfig, expectResponseCode } from '../../api/utils'
 import { Branding } from '../common/branding/branding'
 import { ForkAwesomeIcon } from '../common/fork-awesome/fork-awesome-icon'
+import { FullMarkdownRenderer } from '../markdown-renderer/full-markdown-renderer'
 import { CoverButtons } from './cover-buttons/cover-buttons'
 import { FeatureLinks } from './feature-links'
-import screenshot from './img/screenshot.png'
+
+export const getFrontPageContent = async (pathname: string): Promise<string> => {
+  const baseUrl: string = window.location.pathname.replace(pathname, '')
+
+  const response = await fetch(baseUrl + '/intro.md', {
+    ...defaultFetchConfig,
+    method: 'GET'
+  })
+  expectResponseCode(response)
+
+  return await response.text()
+}
```

Comment:
this should move into the api folder

Reply:
good point

Reply:
Hm, is the intro page content something for the private API in general? I mean it isn't a special pad but just markdown content.

Reply:
Now that you say.. it's no api. It's a file from the public directory.

Reply:
That sounds reasonable.

Reply:
here you go